CONTRIBUTION.md

```diff
@@ -0,0 +1,77 @@
+<h1 align="center">Contributing to The Project </h1>
+
+<p align="center">
+  Welcome to the NotesJar repository! We appreciate your interest in contributing.
+</p>
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+- [Code Style](#code-style)
+- [Reporting Bugs](#reporting-bugs)
+- [Feature Requests](#feature-requests)
+- [Contribution Process](#contribution-process)
+- [Pull Request Guidelines](#pull-request-guidelines)
+- [License](#license)
+
+## Getting Started
+
+1. Fork the repository.
+2. Clone the forked repository to your local machine:
+    ```bash
+    git clone https://github.com/your-username/[project-name].git 
+    ```
+3. Create a new branch for your feature or bug fix:
+    ```bash
+    git checkout -b feature-name
+    ```
+4. Make your changes and commit them with a clear and concise message:
+    ```bash
+    git add .
+    git commit -m "Your clear and concise message"
+    ```
+5. Push your changes to your fork:
+    ```bash
+    git push origin feature-name
+    ```
+6. Open a pull request, referencing any relevant issues.
```

Comment:
Remove the word `any`.

Mention these:

The PR should contain about:
- the problem or issue tag
- the solution approach you have made
- any big file changes
- any bug/issue that can arise from the changes.